### DISCUSSION TITLE
restoring my project using the back up file

### DISCUSSION THREAD

**Discussion by batoolfaran** [supabase/supabase#29764] [Questions]
I am trying to restore my project that using psql tool on pgAdmin 4, this is the code for psql I have psql -d "postgresql://postgres.[PROJECT-REF]:[YOUR-PASSWORD]@aws-0-[REGION].[pooler.supabase.com:5432/postgres](http://pooler.supabase.com:5432/postgres)" -f /path/to/your/file. I created a new project then went to tools-psql tools and tried to fill the requirements of the command but when I run it it gives me an error and I'm not sure if this is the right way to do this? also my back up file I downloaded it from supabase back up after my project was paused. can any one guide me to the right way?

**Comment by olirice** [comment]
There's a new section in the docs https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#restoring-a-downloaded-backup-locally
explaining how to restore your database in a local postgres instance. From that you can copy your data and schema across migration files in a new supabase CLI project and re-upload to a Supabase hosted project

let me know if you have any trouble

**Comment by batoolfaran** [comment]
I was able to restore my project successfully, then I also created a dump file actually they are two one ending with .dump and the other with .sql. now I'm having trouble with migrating the file back to supabase using SQL shell (psql), I have tried multiple command few produced an error few did not but nothing happened either. I followed this resource https://aiherrera.medium.com/transfer-your-local-postgresql-database-to-supabase-4e46ab4474c1, then I tried to follow the supabase documentations in this link https://supabase.com/docs/guides/platform/migrating-to-supabase/postgres?queryGroups=migrate-method&migrate-method=cli#retrieve-credentials, but it was very confusing and I could not figure out where to put or where to find these things, if you could guide me it would be much appreciated. I have attached the command from the first resource.
![code](https://github.com/user-attachments/assets/a2e1cb71-d871-4080-8e69-c17cc2b0d1d9)
